apps/internal/base/internal/storage/storage.go

```diff
@@ -443,6 +424,68 @@ func (m *Manager) writeAppMetaData(AppMetaData AppMetaData) error {
 	return nil
 }
 
+func (m *Manager) RemoveAccount(account shared.Account, clientID string) error {
+	if err := m.removeRefreshTokens(account.HomeAccountID, account.Environment, clientID); err != nil {
+		return err
+	}
+	if err := m.removeAccessTokens(account.HomeAccountID, account.Environment); err != nil {
+		return err
+	}
+	if err := m.removeIDTokens(account.HomeAccountID, account.Environment); err != nil {
+		return err
+	}
+	if err := m.removeAccounts(account.HomeAccountID, account.Environment); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (m *Manager) removeRefreshTokens(homeID string, env string, clientID string) error {
+	m.contractMu.Lock()
+	defer m.contractMu.Unlock()
+	for key, rt := range m.contract.RefreshTokens {
+		if rt.HomeAccountID == homeID && rt.Environment == env {
+			if rt.ClientID == clientID || rt.FamilyID != "" {
```

Comment:
please add comments to explain the logic

Reply:
Added in https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/215/commits/16a8e58fbe4c78cd9c807cb83eae2fa35cb07ccb